bin/apps.bash

```diff
@@ -75,34 +71,75 @@ apps_sc() {
     # While it would be nice to have some template validation before `helmfile apply`,
     # at least Helmfile does "just in time" template validation. Not as nice,
     # but feels good enough until we figure out something smarter.
-    #
+    #    #[ "$1" != "--skip-template-validate" ] && template_validate_sc
+
     #[ "$1" != "--skip-template-validate" ] && template_validate_sc
-    apps_run_sc
+
+    if [ "$#" -gt 0 ] && [ "${*: -1}" = "sync" ]; then
```

Comment:
Since we can be sure that "sync" are on position 3 I would rather have `$3` instead of this. 
Less confusion.